src/commonMain/kotlin/com/github/ajalt/colormath/Constants.kt

```diff
@@ -0,0 +1,26 @@
+package com.github.ajalt.colormath
+
+/**
+ * Reference XYZ illuminant tristimulus values.
+ *
+ * See http://www.brucelindbloom.com/index.html?Eqn_ChromAdapt.html
+ */
+object Illuminant {
```

Comment:
```suggestion
internal object Illuminant {
```

Reply:
Let's make all of these constants internal. We might want to support configurable illuminants, but for now we don't want to add these to the public API.